definitions-terms.md

```diff
@@ -0,0 +1,108 @@
+# Definitions and Terms
+
+A collection of definitions and terms used within this repository.
+
+## TOC
+
+* [Fingerprint](#fingerprint)
+* [Fully Qualified Artifact Reference](#registry-fully-qualified-reference)
+* [Multi-Tenant Registry](#multi-tenant-registry)
+* [OCI Annotations](#oci-annotations)
+* [OCI Descriptor](#oci-descriptor)
+* [OCI Index](#oci-index)
+* [OCI Manifest](#oci-manifest)
+* [Registry](#registry)
+* [SBoM](#sbom)
+* [Subject](#subject)
+* [Tag](#tag)
+
+## Artifact / Image
+
+Artifacts are a generalization of how images are stored in an instance of the [OCI Distribution Spec][oci-distribution-spec]. An [OCI Image][oci-image] is a type of [OCI Artifact][oci-artifact]. When referencing artifacts, within a [fully qualified reference](#registry-fully-qualified-reference), the artifact is right most element of the namespace, combined with the `:tag`.  
```

Comment:
> the artifact is right most element of the namespace

This isn't really accurate.  The right-most element of the name is still just part of the name, and can resolve to anything the registry stores.  This could be an index, an image manifest, or an artifact.

We should probably include a definition for a name, referencing the [distribution spec](https://github.com/opencontainers/distribution-spec/blob/master/spec.md#overview):

> The rules for a repository name are as follows:
>
> 1. A repository name is broken up into *path components*. A component of a repository name MUST begin with one or more lowercase alpha-numeric characters. Subsequent lowercase alpha-numeric characters are OPTIONAL and MAY be separated by periods, dashes or underscores. More strictly, it MUST match the regular expression `[a-z0-9]+(?:[._-][a-z0-9]+)*.`
> 2. If a repository name has two or more path components, they MUST be separated by a forward slash ("/").
> 3. The total length of a repository name, including slashes, MUST be less than 256 characters.

Reply:
This looks like the definition I wrote for repo, below.

Reply:
Lets have this conversation on Monday.